independent-publisher-connectors/FreeAgent/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1847 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "FreeAgent",
+    "contact": {
+      "name": "Nirmal",
+      "url": "https://nkdews.me",
+      "email": "nirmalk.web@gmail.com"
+    },
+
```

Comment:
@nk-gears Please remove any unwanted spaces from swagger.

Reply:
@sriyen-msft Fixed and Committed.